app/app.go

```diff
@@ -413,7 +415,7 @@ func (app *Application) ExportAppStateAndValidators(forZeroHeight bool, jailWhit
 
 		for ; kvStoreReversePrefixIterator.Valid(); kvStoreReversePrefixIterator.Next() {
 			addr := sdk.ValAddress(kvStoreReversePrefixIterator.Key()[1:])
-			validator, found := app.StakingKeeper.GetValidator(context, addr)
+			validator, found := app.StakingKeeper.GetLiquidValidator(context, addr)
```

Comment:
why only getLiquidValidator?

Reply:
GetLiquidValidator is an in-place replacement for GetValidator, it uses upgraded `` that allows to read extra fields that are related to ls :)

See the implementation difference between two:
https://github.com/persistenceOne/persistence-sdk/blob/3773b08f31a14f04c6f804851449b55979997bab/x/lsnative/staking/keeper/validator.go#L16-L39
(only `MustUnmarshalValidator`)

Reply:
type `Validator` from the forked version is extended with these fields:
![Uploading Screenshot 2023-04-04 at 12.48.20.png…]()

Reply:
type `Validator` from the forked version is extended with these fields:
<img width="824" alt="Screenshot 2023-04-04 at 12 48 20" src="https://user-images.githubusercontent.com/477998/229769315-421289c3-a717-4de1-ab82-66186f7e1813.png">

Reply:
got it thanks!